changelog.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v0.4.0 | 2019-12-05
+
+* Add calibration data as attribute of force channels.
+* Take into account slicing of channels when returning force calibration data.
+* Fixed slicing behaviour when slicing continuous time channels within a timestep.
+* Export pixel size and dwell time to TIFFs exported with pylake.
```

Comment:
"with pylake" is not necessary. It's assumed since this is the pylake changelog.

Reply:
👍